toke.c

```diff
@@ -5515,6 +5515,64 @@ yyl_hyphen(pTHX_ char *s)
     }
 }
 
+static int
+yyl_subproto(pTHX_ char *s, CV *cv)
+{
+    STRLEN protolen = CvPROTOLEN(cv);
+    const char *proto = CvPROTO(cv);
+    bool optional;
+
+    if (!SvPOK(cv))
```

Comment:
maybe check it and return earlier?

Reply:
Yeah, this commit actually changed the apparent order of the `SvPOK()`, `CvPROTOLEN()`, and `CvPROTO()` operations — good spot, thank you! I'll squash a change in and force-push.